buoy_gazebo/src/MooringForce/MooringForce.cpp

```diff
@@ -202,12 +211,36 @@ void MooringForce::Configure(
   gzdbg << "Effective radius set to beyond " << this->dataPtr->effectiveRadius
     << std::endl;
 
+  if (_sdf->HasElement("chain_length"))
+  {
+    this->dataPtr->L = _sdf->Get<double>(
+      "chain_length", this->dataPtr->L).first;
+    gzdbg << "Mooring chain length set to " << this->dataPtr->L
+      << std::endl;
+  }
+
   // Find necessary model links
   if (this->dataPtr->FindLinks(_ecm)) {
     this->dataPtr->UpdateVH(_ecm);
   }
 
   this->dataPtr->B.resize(1U);
+
+  // debug print throttle, default 1Hz
+  {
+    double rate(1.0);
+    if (_sdf->HasElement("debug_print_rate"))
+    {
+      rate = _sdf->Get<double>(
+        "debug_print_rate", this->dataPtr->L).first;
+      gzdbg << "Debug print rate set to " << this->dataPtr->L
```

Comment:
Typos from copy-paste
```suggestion
        "debug_print_rate", rate).first;
      gzdbg << "Debug print rate set to " << rate
```